### PR TITLE
feat(web): Tweak little cases bots page

### LINKED ISSUE #221: Update the Little Cases page
Once we've got our modeling figured out in #220, we should redo the Little Cases page:

 - The Suggest a Bot form becomes a button on the page that leads to /little-cases/suggest-a-bot/.
 - The page itself explains what little bots are, and provides links to all of the little bots we've got.
 - Each Little Bot gets a URL that can describe its purpose (we'll need a new DB field for this, I guess) and the cases it follows.
    - The URLs could be /little-cases/@handle, where the handle that works is the one from #220 that doesn't have an alias.

Thoughts?

### PR DESCRIPTION
**Pull request by ERosendo**
This PR tweaks the page for Little Cases Bots and adds a template to show the purpose of each little cases bot and the cases it follows. This PR fixes #221.

This PR also removes the form to suggest a bot from the Little Cases page and uses a button instead. Here's a screenshot of the new page for the little cases bots.

<p align="center">
  
</p>

Here's also a screenshot of the page to show details about each little case bot:

<p align="center">
  
</p>

In this new page for details, We include an overview of the bot, buttons to follow the channels linked to the bot, and the list of cases it follows. 

This PR depends on #247 (We should merge that one first).

### CODE REVIEW

File: bc/channel/models.py [hunk 1]
```diff
@@ -23,10 +28,19 @@ class Group(AbstractDateTimeModel):
         related_name="groups",
         blank=True,
     )
+    slug = models.SlugField(
+        help_text="A generated path for this item", null=True
+    )
+    overview = models.TextField(
+        help_text="Short description of the purpose of this group", null=True
+    )
```
Comment (mlissner) [inline bc/channel/models.py:36]: String fields shouldn't ususally be nullable, right? They should be blankable instead?

File: bc/web/templates/little-cases/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{% extends "post.html" %}
+{% block title %}Little Cases Bots{% endblock %}
+
+{% block post_title %}Little Cases Bots{% endblock %}
+
+{% block post_content %}
+  <p class="lead">We are developing a system where you can create and maintain your own bots for Twitter or Mastodon.</p>
+  <p>You'll give us permission to post on your behalf and we'll send updates for cases you subscribe to on CourtListener.</p>
+  <p>Imagine a bot that keeps you and the public updated about important topics like intellectual property, immigration, or online privacy.</p>
+  <p>If there's a legal topic you follow and if you want to curate a bot for that topic, let us know using the following button. We'll get back to you as soon as possible with a plan for action.</p>
```
Comment (mlissner) [inline bc/web/templates/little-cases/index.html:10]: I'll do some tweaks on this before we merge.

**Review by mlissner — CHANGES_REQUESTED** [review]
Looks good to me. A few small comments, and it needs a rebase, please (there are changes here from earlier PRs).

File: bc/channel/models.py (hunk 1 above)
Comment (ERosendo) [reply bc/channel/models.py]: You're right! I'm going to remove the null argument for these fields.

**Review by mlissner — APPROVED** [review]

**Pull request merged**